examples/misc/get_all_videos_and_images.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This code example gets information about all video and image files."""
+
+
+import argparse
+import sys
+from google.ads.google_ads.client import GoogleAdsClient
+from google.ads.google_ads.errors import GoogleAdsException
+
+_DEFAULT_PAGE_SIZE = 1000
+
+
+def main(client, customer_id, page_size):
+    """Main method, to run this code example as a standalone application."""
+    ga_service = client.get_service('GoogleAdsService', version='v3')
+
+    # Creates a query that will retrieve all video and image files.
+    query = ('SELECT media_file.id, media_file.name, media_file.type '
+             'FROM media_file ORDER BY media_file.id')
+
+    # Issues a search request by specifying page size.
+    results = ga_service.search(customer_id, query=query, page_size=page_size)
+
+    media_type_enum = client.get_type('MediaTypeEnum', version='v3').MediaType
+
+    try:
```

Comment:
As you also include many comments here, could you include one more here too?
"Iterates over all rows and prints the information about each media file."

Reply:
Yup sure. Added comment here.